recipe/meta.yaml

```diff
@@ -27,14 +27,17 @@ requirements:
         - zlib
 
 test:
+    requires:
+        - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
Ping @jakirkham and @pelson I no longer remember what was the decision here, but I needed to add this avoid getting the unresolved dependencies.